content/blog/2018/06/2018-06-19-new-design-digest.adoc

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: "New design and UX digest"
```

Comment:
IMHO the name should explicitly include "New login page design and extensibility" or so